po/eu.po

```diff
@@ -54,115 +57,69 @@ msgstr ""
 "aukeratutako koloretik abiatuta."
 
 #: data/com.github.finefindus.eyedropper.metainfo.xml.in.in:14
-msgid "Available Formats:"
-msgstr "Formatu erabilgarriak:"
+#: src/widgets/about_window.rs:82
+msgid "Features"
+msgstr "Características"
 
 #: data/com.github.finefindus.eyedropper.metainfo.xml.in.in:16
-msgid "Hex"
-msgstr "Hex"
+#: src/widgets/about_window.rs:86
+msgid "Pick a Color"
+msgstr "Aukeratu colore bat"
 
 #: data/com.github.finefindus.eyedropper.metainfo.xml.in.in:17
-#: src/widgets/preferences.rs:311
-msgid "RGB"
-msgstr "RGB"
+#: src/widgets/about_window.rs:87
+msgid "Enter a color in Hex-Format"
+msgstr "Sartu kolore bat formatu hexadezialean"
 
 #: data/com.github.finefindus.eyedropper.metainfo.xml.in.in:18
-#: src/widgets/preferences.rs:319
-msgid "HSL"
-msgstr "HSL"
+#: src/widgets/about_window.rs:88
+msgid "Parse RGBA/ARGB Hex-Colors"
+msgstr "Aztertu RGBA/ARGB kolore hexadezigaitzak"
 
 #: data/com.github.finefindus.eyedropper.metainfo.xml.in.in:19
-#: src/widgets/preferences.rs:328
-msgid "HSV"
-msgstr "HSV"
+#: src/widgets/about_window.rs:89
+msgid "View colors in Hex, RGB, HSV, HSL, CMYK, XYZ and CIE-Lab format"
+msgstr "Koloreak honela ikus ditzakezu: Hex, RGB, HSV, HSL, CMYK, XYZ eta CIE-Lab Format"
+
+#: data/com.github.finefindus.eyedropper.metainfo.xml.in.in:14
+msgid "Available Formats:"
+msgstr "Formatu erabilgarriak:"
 
 #: data/com.github.finefindus.eyedropper.metainfo.xml.in.in:20
-#: src/widgets/preferences.rs:337
-msgid "CMYK"
-msgstr "CMYK"
+#: src/widgets/about_window.rs:90
+msgid "Customize which formats appear as well as their order"
+msgstr ""
+"Pertsonalizatu zer formatu agertzen diren, bai eta haien ordena ere."
```

Comment:
The translation end with a dot, whereas the source doesn't. Although I should probably change that, so it's ok.

Reply:
Okay. I've removed the dot.